resource_processor/shared/logging.py

```diff
@@ -96,9 +106,13 @@ def shell_output_logger(console_output: str, prefix_item: str, logger: logging.L
     """
     logger.log(logging_level, prefix_item)
 
-    if console_output is None:
+    if (console_output is None or len(console_output) == 0):
```

Comment:
can this be collapsed to just...

```
if console_output:
```

given that both None and "" are falsey ?

Reply:
done